Makefile

```diff
@@ -368,7 +368,13 @@ docsgen-openrpc-worker: docsgen-openrpc-bin
 .PHONY: docsgen docsgen-md-bin docsgen-openrpc-bin
 
 gen: type-gen method-gen docsgen api-gen
+	@echo ">>> IF YOU'VE MODIFIED THE CLI, REMEMBER TO ALSO MAKE docsgen-cli"
```

Comment:
Is doscgen-cli slow? Why not just always run it?

Reply:
It's not in gen because I'm not sure how to force make to execute one target after another when running in parallel without making one target depend on another - docsgen-cli depends on lotus binaries, which can't build if we're in the middle of generating api code, but we don't want the lotus build to depend on gen